lib/seattleflu/db/cli/command/reportable_conditions.py

```diff
@@ -0,0 +1,322 @@
+        where identifier = %s and lineage = %s
+        """, (identifier, lineage))
+
+    if reportable_condition:
+        LOG.info(f"Found sample «{identifier}», organism of type «{lineage}»")
+    else:
+        data = {
+            "identifier": identifier,
+            "lineage": lineage,
+            "site": site,
+        }
+
+        reportable_condition = db.fetch_row("""
+            insert into shipping.reportable_condition (identifier, lineage, site)
+                values (%(identifier)s, %(lineage)s, %(site)s)
+            returning identifier, lineage, site
+            """, data)
+
+        LOG.info(f"Created sample «{identifier}», organism of type «{lineage}»")
+
+    return reportable_condition
+
+
+@reportable_conditions.command("notify")
+
+@click.option("--dry-run", "action",
+    help        = "Only go through the motions of changing the database (default)",
+    flag_value  = "rollback",
+    default     = True)
+
+@click.option("--prompt", "action",
+    help        = "Ask if changes to the database should be saved",
+    flag_value  = "prompt")
+
+@click.option("--commit", "action",
+    help        = "Save changes to the database",
+    flag_value  = "commit")
+
+def notify(*, action: str):
+    """
+    Contains some hard-coded
+    """
+    LOG.debug(f"Starting the reportable conditions notification routine, revision {REVISION}")
+
+    db = DatabaseSession()
+
+    SLACK_WEBHOOK_URL = os.environ['SLACK_WEBHOOK_URL']  # Testing only
+    SLACK_WEBHOOK_REPORTING_GENERAL = os.environ['SLACK_WEBHOOK_REPORTING_GENERAL']
+    SLACK_WEBHOOK_REPORTING_CHILDRENS = os.environ['SLACK_WEBHOOK_REPORTING_CHILDRENS']
+
+    childrens_sites = get_childrens_sites(db)
+
+    # Fetch and iterate over reportable condition records that aren't processed
+    #
+    # Rows we fetch are locked for update so that two instances of this
+    # command don't try to process the same reportable condition records.
+    LOG.debug("Fetching unprocessed reportable conditions records")
+
+    reportable_conditions = db.cursor("reportable_conditions")
+    reportable_conditions.execute("""
+        select reportable_condition_id as id,
+          identifier, lineage, site
+          from shipping.reportable_condition
+         where not processing_log @> %s
+         order by id
+           for update
+        """, (Json([{ "revision": REVISION }]),))
+    processed_without_error = None
+
+    try:
+        for record in reportable_conditions:
+            with db.savepoint(f"reportable conditions record {record.id}"):
+                LOG.info(f"Processing reportable condition record {record.id}")
+
+                url = SLACK_WEBHOOK_REPORTING_GENERAL \
+                    if record.site not in childrens_sites \
+                    else SLACK_WEBHOOK_REPORTING_CHILDRENS
+
+                # TODO replace SLACK_WEBHOOK_URL with url
+                response = send_slack_post_request(record, SLACK_WEBHOOK_URL)
+
```

Comment:
`SLACK_WEBHOOK_URL` will be replaced with `url` when this is pushed to production. The testing url is used meanwhile to avoid channel notification spamming.